scripts/src/javatests/org/oppia/android/scripts/label/AccessibilityLabelCheckTest.kt

```diff
@@ -0,0 +1,300 @@
+  }
+
+  @Test
+  fun testAccessibilityLabel_multipleFailures_logsShouldBeLexicographicallySorted() {
+    val testContent1 =
+      """
+      <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="org.oppia.android">
+          <activity
+              android:name=".app.FourthTempActivity" />
+          <activity
+              android:name=".app.ThirdTempActivity" />
+          <activity
+              android:name=".app.FirstTempActivity" />
+          <activity
+              android:name=".app.SecondTempActivity"
+              android:label="@string/administrator_controls_title2" />
+      </manifest>
+      """.trimIndent()
+    val testContent2 =
+      """
+      <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="org.oppia.android.splash">
+          <activity
+              android:name=".FirstSplashActivity"
+              android:label="@string/administrator_controls_title1" />
+          <activity
+              android:name=".SecondSplashActivity" />
+      </manifest>
+      """.trimIndent()
+    tempFolder.newFolder("testfiles", "app", "src", "main")
+    tempFolder.newFolder(
+      "testfiles", "app", "src", "main", "java", "org", "oppia", "android", "splash"
+    )
+    val appManifestPath = "app/src/main/AndroidManifest.xml"
+    val splashManifestPath = "app/src/main/java/org/oppia/android/splash/AndroidManifest.xml"
+    val appManifestFile = tempFolder.newFile("testfiles/$appManifestPath")
+    val splashManifestFile = tempFolder.newFile("testfiles/$splashManifestPath")
+    appManifestFile.writeText(testContent1)
+    splashManifestFile.writeText(testContent2)
+
+    val exception = assertThrows(Exception::class) {
+      main(retrieveTestFilesDirectoryPath(), appManifestPath, splashManifestPath)
+    }
+
+    assertThat(exception).hasMessageThat().contains(
+      ACCESSIBILITY_LABEL_CHECK_FAILED_OUTPUT_INDICATOR
+    )
+    val firstAppActivityPath = "app/src/main/java/org/oppia/android/app/FirstTempActivity"
+    val thirdAppActivityPath = "app/src/main/java/org/oppia/android/app/ThirdTempActivity"
+    val fourthAppActivityPath = "app/src/main/java/org/oppia/android/app/FourthTempActivity"
+    val splashActivityPath = "app/src/main/java/org/oppia/android/splash/SecondSplashActivity"
+    val failureMessage =
+      """
+      Accessibility label missing for Activities:
+      - ${retrieveTestFilesDirectoryPath()}/$firstAppActivityPath
+      - ${retrieveTestFilesDirectoryPath()}/$fourthAppActivityPath
+      - ${retrieveTestFilesDirectoryPath()}/$thirdAppActivityPath
+      - ${retrieveTestFilesDirectoryPath()}/$splashActivityPath
+      
+      $failureNotePartOne
+      $failureNotePartTwo
+      """.trimIndent()
+    assertThat(outContent.toString().trim()).isEqualTo(failureMessage)
+  }
+
+  @Test
+  fun testAccessibilityLabel_AccessibilitylabelNotDefinedForExemptedActivity_checkShouldPass() {
```

Comment:
```suggestion
  fun testAccessibilityLabel_accessibilityLabelNotDefinedForExemptedActivity_checkShouldPass() {
```

Note capitalization.

Reply:
Done.